posts/dialog.md

```diff
@@ -0,0 +1,12 @@
+feature: dialog
+status: use
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
```

Comment:
I've only recently inherited the polyfill - it seems like the original authors didn't list browser compatibility. I'll have to do some testing to confirm, but I think it's more appropriate to list the supported browsers on the `dialog-polyfill` page (none of the other items on html5please seem to list polyfill requirements).
How about I offer to make sure the compatibility list on our side is up-to-date? :-)